.travis.yml

```diff
@@ -13,3 +13,6 @@ matrix:
     - choco install openssl.light
     script:
     - $WORKSPACE/scripts/windows_build.sh
+branches:
+  only:
+  - master
```

Comment:
add newline?

Reply:
Not required for this config. 
However, added.

Reply:
Seems github has a problem with unterminated files.